src/main/resources/templates/journals/create.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org" xmlns:sec="http://www.thymeleaf.org/extras/spring-security">
+<head  th:replace="partials/elements :: head(title='FilmSage Index')">
+    <meta charset="UTF-8"/>
+    <title>Create a Journal</title>
+</head>
+<body>
+<nav th:replace="partials/elements :: navbar"></nav>
+<form th:action="@{/journals/create}" th:method="POST" th:object="${'newJournal'}">
+    <div class="form-group">
+        <label for="title">Title</label>
+        <input type="text" class="form-control" id="title" name="title" th:field="*{'title'}">
+    </div>
+
+    <div class="form-group">
+        <label for="body">Body</label>
+        <textarea class="form-control" id="body" rows="3" name="body" th:field="*{'body'}"></textarea>
+        <input type="hidden" th:field="*{'user'}" id="user">
```

Comment:
drop this for now. we need to get user session working and retrievable and the user is not part of the model in the get request mapping anyway so this will cause a null pointer exception/503. where is this page's get request mapping in the controller? this only has a post mapping.